patches/@bazel+typescript+2.0.0-rc.0.patch

```diff
@@ -0,0 +1,98 @@
+diff --git a/node_modules/@bazel/typescript/internal/ts_project.bzl b/node_modules/@bazel/typescript/internal/ts_project.bzl
```

Comment:
What does this patch do?

Reply:
See https://github.com/bazelbuild/rules_nodejs/pull/1898#issuecomment-646666257 for context

Reply:
Can you add a README in this folder to explain for posterity?

Reply:
yup will do